tests/events_test.py

```diff
@@ -191,3 +191,25 @@ def observer(new_event: Event, old_event: Optional[Event] = None) -> None:
         with patch("zino.events.EVENT_DUMP_DIR", tmp_path):
             events._delete(event)
         assert observer.called
+
+    def test_delete_should_not_delete_open_event(self):
+        events = Events()
+        event = events.get_or_create_event("foobar", None, ReachabilityEvent)
+        events.commit(event)
+
+        events._delete(event)
+
+        index = EventIndex("foobar", None, ReachabilityEvent)
+        assert events._events_by_index.get(index)
+        assert events.events.get(event.id)
+
```

Comment:
Maybe explicitly set EventState.OPEN (or ad a comment that it by default is open) to contrast better with the next test.